openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -84,10 +84,8 @@ <h2 class="hd hd-3 page-title">${course.display_name_with_default}</h2>
             % endif
         </div>
         <aside class="page-content-secondary course-sidebar">
-            % if show_proctoring_info_panel:
-                <div class="proctoring-info-panel"
-                     data-course-id="${course_key}" data-username="${username}"></div>
-            % endif
+            <div class="proctoring-info-panel"
```

Comment:
I think this value can also be removed from `course_home.py`

Reply:
Not in this PR. This PR is specifically removing that flag from the frontend. This way, the backend can send the variable but the frontend just won't use it.  
Then when I remove the flag from the `course_home.py` file, the frontend will be oblivious of that change.